lua/lazyvim/util/init.lua

```diff
@@ -200,6 +200,18 @@ function M.toggle_number()
   end
 end
 
+function M.toggle_autopairs()
+  if package.loaded["mini.pairs"] then
+    if vim.g.minipairs_disable then
+      vim.g.minipairs_disable = false
+      Util.info("Enabled auto pairs", { title = "Option" })
+    else
+      vim.g.minipairs_disable = true
+      Util.warn("Disabled auto pairs", { title = "Option" })
```

Comment:
Maybe `INFO` level would be better here since this isn't unexpected or an error? The user is manually toggling the state/functionality.

Reply:
I was just following the same pattern as `toggle_number` and `toggle_diagnostics`. They both use `warn` when disabling.

Reply:
Ah ok, didn't realize that was the precedent.